beets/library.py

```diff
@@ -1601,14 +1602,20 @@ class Library(dbcore.Database):
     def __init__(
         self,
         path="library.blb",
-        directory="~/Music",
+        directory: str | None = None,
         path_formats=((PF_KEY_DEFAULT, "$artist/$album/$track $title"),),
         replacements=None,
     ):
         timeout = beets.config["timeout"].as_number()
         super().__init__(path, timeout=timeout)
 
-        self.directory = bytestring_path(normpath(directory))
+        if directory is not None:
```

Comment:
Oh, just noticed now you could possibly have just a single condition here, I think:

```python
        if directory is None:
            directory = platformdirs.user_music_path()
        self.directory = bytestring_path(directory)
```

Reply:
Yeah, I had no clue which of the `util` functions to use.  If this one works, then I'll use it, thanks.